packages/blocks/drawing-canvas/block-schema.json

```diff
@@ -1,12 +1,9 @@
 {
   "type": "object",
   "properties": {
-    "document": {
+    "documentString": {
```

Comment:
How about `serializedDocument`? `String` suffix is a bit too technical.

Reply:
Sounds good, will update